plugins/outputs/nats/README.md

```diff
@@ -74,6 +74,13 @@ to use them.
     # name = ""
     # subjects = []
 
+    ## Use asynchronous publishing for higher throughput, but note that it does not guarantee order within batches.
+    # async_publish = false
+
+    ## Timeout for wating on acknowledgement on asynchronous publishing
+    ## String with valid units "ns", "us" (or "µs"), "ms", "s", "m", "h".
```

Comment:
This is true for all times and we should not add this over and over in the configs...
```suggestion
```